Style onboarding screen

linyh0207 opened the pull request:
1. Add logo
2. Update the icons
3. Style the login and sign up modals

Not sure how to cite the icons. Only put it as a comment below icons for now.

Michael-Xie commented:
- the login and register button can be pushed up a bit, it feels like they are going to fall off the screen; distance between Sign Up and bottom of screen can be bigger
- the detail of the snap pea in the logo could be hard to see, might consider something simpler or bigger
- for Help carousel item, how about "Help each other reduce food wastes and food costs."?
- For register modal, the error message should show when email is blank

Michael-Xie commented:
for modal style, how about having the borders round like your styled button?

linyh0207 commented:
1. Adjusted the margin for login and sign up buttons
2. Made the logo bigger. I feel like the focus should be the name not the logo. I replaced the camera icon to a focus icon from your sketch because we already using the camera icon for the first carousel item.
3. Updated the carousel item descriptions.
4. Fixed the error message for sign up modal, it will show 'email address is empty!' when it's empty. When user starts texting, it will identify if it includes @.
5. Make both of the login and sign up modals rounded corners.
